internal/dataplane/parser/errors_test.go

```diff
@@ -0,0 +1,82 @@
+package parser_test
+
+import (
+	"testing"
+
+	"github.com/sirupsen/logrus"
+	"github.com/sirupsen/logrus/hooks/test"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/kong/kubernetes-ingress-controller/v2/internal/dataplane/parser"
+	kongv1 "github.com/kong/kubernetes-ingress-controller/v2/pkg/apis/configuration/v1"
+)
+
+const someValidTranslationErrorReason = "some valid reason"
+
+var someValidTranslationErrorCausingObjects = []client.Object{&kongv1.KongIngress{}, &kongv1.KongPlugin{}}
+
+func TestTranslationError(t *testing.T) {
+	t.Run("is_created_and_returns_reason_and_causing_objects", func(t *testing.T) {
```

Comment:
if you use space, it will be replaced to `_` by default

Reply:
Yup, but having `_` explicitly in the test name makes it easily searchable when tests fail. I can change to spaces if you think that's important, but IMO it's still the same level of readability + the advantage of searchability, WDYT?

Reply:
I'm ok with that

Reply:
I'm also OK with that but please note that this was it won't be consistent with what we have :)

Reply:
After giving it more thought I agree consistency wins, even if the approach I used has some advantage. I guess that is something that we could discuss as a coding guideline, but let's leave it for now: cbde1f0